hebrew.html

```diff
@@ -119,6 +119,22 @@ <h1>
       </div>
     </div>
 
+    <div class = "covid">
+      <div class = "covid-content">
+        <h1>
+          COVID-19
+        </h1>
+        <div class = "covid-container">
+          <p>
+            Qint
+            מספקת הגנה מקסימלית עבור לקוחות ועובדים בבית העסק, על ידי ניהול תור ללא מגע תוך כדי שמירה על מרחק בין לקוחות ועל תורם, ובנוסף מאפשרת בקרה על איזון עומסים בכל זמן נתון.
+          </p>
+        <p><img src ="assets/Covid-RTL.png" class="imageCovid"/></p>
```

Comment:
Instead of serving an entirely new image, you can `transform: scaleX(-1);` the usual one to flip it.